lib/core/src/lib/viewer/components/img-viewer/img-viewer.component.html

```diff
@@ -58,7 +58,7 @@
 
     </adf-toolbar>
 
-    <adf-toolbar class="adf-secondary-toolbar" *ngIf="!readOnly && isEditing">
+    <adf-toolbar class="adf-secondary-toolbar" *ngIf="isEditing && (!readOnly || allowRotateInReadOnly)">
```

Comment:
Why do we want to display the cancel and the save buttons if we in readonly mode?

Reply:
This is comes from product side. [MMF](https://hyland.atlassian.net/browse/HXPM-823)

<img width="546" alt="Screenshot 2024-06-27 at 7 19 16 PM" src="https://github.com/Alfresco/alfresco-ng2-components/assets/32226301/96fb171b-b012-465d-bc7d-d5f88b8096be">

Reply:
Then, if we want to make it permanent and save the changes, we will not be in readOnly mode, right?

Reply:
Currently, the adf-viewer operates in two modes:

Read-only: No action toolbar or button toolbar is visible.
Editable: Both toolbars are visible, but the buttons become visible only after starting an action like rotating or cropping.

Our MMF requires activating only the rotate button with the save feature while keeping the viewer in read-only mode. We cannot directly make the viewer editable, as this would also activate cropping, which is not desired in this scenario.

Reply:
Enabling "the save feature" seems just like the opposite of being read-only mode, is it not? From what I read, it seems to me that we want to be in editable mode, but also to be able to control which actions can be displayed in this mode, i.e., we'd like to disable the crop action for hxviewer which would use the adf-viewer to edit a document. Is this the case?

Reply:
You're right that enabling the save feature seems contrary to the read-only mode. Instead, we could have the viewer in editable mode but control which actions are available, which would work and be meaningful. However, I am concerned about how this change might impact existing applications that use the viewer in editable mode.

Reply:
There's probably more than one way to achieve this. A way would be to add a configuration property to control which actions are enabled on edit mode and, by default, it would have the current configuration, to avoid introducing breaking changes. On HxViewer, this property could be changed to only show the rotate action.

Reply:
Please check [here] (https://github.com/Alfresco/alfresco-ng2-components/pull/9873#discussion_r1657456365)